spec/spec_helper.rb

```diff
@@ -89,25 +93,14 @@ class ::App < Rails::Application
   config.before :each do
     evaljs("var window = this;", true)
 
-    def inspectify(value)
-      case value
-      when V8::Array
-        value.map do |v|
-          inspectify(v)
-        end
-      when V8::Object
-        value.to_h.map do |k,v|
-          [k, inspectify(v)]
-        end.to_h
-      when String, nil, Integer, FalseClass, TrueClass
-        value
-      else
-        raise "wtf #{value.class}?"
+    if defined?(JRUBY_VERSION)
+      jscontext[:log] = lambda do |context, value|
+        puts value
```

Comment:
Is it just working now and we don't need to format manually?

Reply:
@bogdan yes, it is benefit from mini_racer - it is doing converting of objects from JS-to-Ruby itself.